rotkehlchen/tests/api/test_data_import.py

```diff
@@ -79,14 +80,34 @@ def test_data_import_cryptocom(rotkehlchen_api_server):
     assert_cryptocom_import_results(rotki)
 
 
+@pytest.mark.parametrize('number_of_eth_accounts', [0])
+def test_data_import_cryptocom_special_types(rotkehlchen_api_server):
+    """Test that the data import endpoint works successfully for cryptocom"""
+    rotki = rotkehlchen_api_server.rest_api.rotkehlchen
+    dir_path = Path(os.path.dirname(os.path.dirname(os.path.realpath(__file__))))
```

Comment:
Here and below you can get current file and parent file relations still using pathlib and not use os.path. An example here: https://github.com/rotki/rotki/blob/8443d1ee87f1ebfcb60c7420a992ecae459706a1/rotkehlchen/globaldb/handler.py#L47